dapps/assets/src/components/DataContainer.ts

```diff
@@ -29,17 +29,19 @@ import { SearchService, DigitalTwin } from '@evan.network/digital-twin-lib';
 
 @Component
 export default class DataContainerComponent extends mixins(EvanComponent) {
-  searchTerm = '';
-
-  search = new SearchService(this.getRuntime());
+  count = 20;
 
   data = [];
 
+  error = false;
```

Comment:
boolean var names should indicate that they are boolean, like hasError, isErroneous